Source/analyze_data.cpp

```diff
@@ -1,33 +1,69 @@
 #include "analyze_data.h"
 
-void ANALYZE_DATA::analyze_data(float32_t* p_raw_data, float32_t* p_data,
-      float32_t* p_autocorr_data, float32_t* p_intensity,
-      uint32_t* p_lag)
+/**
+ * Initializing the extern filter-variables
+ */
+float32_t state_coefficients[4 * num_stages] = 
+{
+        0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0
+};
+
+float32_t filter_coefficients[5 * num_stages] =
+{
+        0.56942484, 0.0, -0.56942484,             /* Numerator filter 1                               */
```

Comment:
Is there any documentation that explains how the filter coefficients are chosen?

Reply:
Is in the headerfile parameters.h

Reply:
See parameters.h

Reply:
Nice :+1: Could you add a reference to that here? Just to make it easy to find for others

Reply:
Done